src/shapes.rs

```diff
@@ -0,0 +1,42 @@
+use std::f64::consts::PI;
+
+use vello::kurbo::{Line, Point, Vec2};
+
+const ARROW_ARM_LENGTH: f64 = 20.0;
+const ARROW_ARM_ANGLE: f64 = 0.2 * PI;
+
+pub fn arrow(from: Point, to: Point) -> [Line; 3] {
+    let backwards_direction = (from.to_vec2() - to.to_vec2()).normalize();
+
+    let clockwise_arm =
+        to + (Mat2::rotate(ARROW_ARM_ANGLE) * backwards_direction * ARROW_ARM_LENGTH);
+    let anticlockwise_arm =
+        to + (Mat2::rotate(-ARROW_ARM_ANGLE) * backwards_direction * ARROW_ARM_LENGTH);
+
+    [
+        Line::new(from, to),
+        Line::new(to, clockwise_arm),
+        Line::new(to, anticlockwise_arm),
+    ]
+}
+
+// TODO: is there something inside of one of the vello libraries
+// that would let us do this without having to implement it ourselves?
+struct Mat2([f64; 4]);
```

Comment:
Would `kurbo::Affine` do what you need?

Reply:
I think so, if I figure out the math math :grin:. It didn't have `Mul` implemented with `Vec2` on the version I had originally used to write this--I'm not sure if it does now.

To me the simplest modeling of this was building a 2D vector which could be added to the point, so I just hacked together something that could do what I needed.

Reply:
https://github.com/linebender/kurbo/issues/360